src/Repository/RiddenCoasterRepository.php

```diff
@@ -356,4 +356,19 @@ public function findUserRatingsForRanking(int $userId): array
 
         return $query->getResult();
     }
+
+    public function getCountRatingsForCoaster(Coaster $coaster): array
```

Comment:
could you transform this method into a one liner like: `return $this->getEntityManager()->createQuery(...)->setParameter(...)->getResult()` ?